packages/core/src/connectivity/scp-cf/index.ts

```diff
@@ -9,7 +9,7 @@ export * from './environment-accessor';
 export * from './environment-accessor-types';
 export * from './resilience-options';
 export * from './token-accessor';
-export * from './xsuaa-service';
+export * from './legacy/xsuaa-service';
```

Comment:
I guess we miss the new `xsuaa-service`.

Reply:
I thought it was internal on purpose. However, I moved the `fetchVerificationkeys` to a separate file and exportet it to avoid breaking changes.